scripts/soundness.sh

```diff
@@ -53,6 +53,12 @@ if git grep --color=never -i "${unacceptable_terms[@]}" -- . ":(exclude)CODE_OF_
 fi
 printf "\033[0;32mokay.\033[0m\n"
 
+# swift-format
+SWIFT_FORMAT_SCRIPT="${here}/run-swift-format.sh"
```

Comment:
shell variables are `lower_snake_case`, `UPPER_CASE_ONES` are reserved for POSIX and other things that are meant to change the behaviour of _other_ processes.

Reply:
The script is already full of it

Reply:
I don't understand the argument. Why would we want to make it worse just because it's already bad?

Reply:
If that's the convention in the NIO repo, then fair enough.

Reply:
So it isn't a POSIX standard but just a convention that NIO follows to some degree?

Reply:
> The name space of environment variable names containing lowercase letters is reserved for applications.

https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html